tests/baseTestAPI.py

```diff
@@ -176,6 +176,8 @@ def setUpClass(cls):
 
     def setUp(self):
         """Set up broker unit tests."""
+        # Repopulate interfaces if needed
+        self.interfaces = InterfaceHolder()
```

Comment:
Is there a reason this is using `InterfaceHolder()` vs `BaseInterface.interfaces` as the broker was using?